src/main/java/org/jenkinsci/plugins/artifactdeployer/ArtifactDeployerEntry.java

```diff
@@ -46,7 +46,7 @@
     private boolean flatten;
     private boolean deleteRemote;
     private boolean deleteRemoteArtifacts;
-    private boolean deleteRemoteArtifactsByScript=false;
+    private transient boolean deleteRemoteArtifactsByScript=false;
```

Comment:
The idea is that this doesn't get stored to disk anymore as the related feature is gone. This depends on https://github.com/jenkinsci/artifactdeployer-plugin/compare/artifactdeployer-0.33...master